app/src/main/java/fr/neamar/kiss/dataprovider/SearchProvider.java

```diff
@@ -16,7 +17,7 @@
 
 public class SearchProvider extends Provider<SearchPojo> {
     private SharedPreferences prefs;
-    private static final LinkedHashMap<String,String> searchProviderUrls = new LinkedHashMap<>();
+    private static final HashMap<String,String> searchProviderUrls = new LinkedHashMap<>();
```

Comment:
Fixed in https://github.com/Neamar/KISS/commit/ead386775043d1aa50192757324fbf96aeedb6e6